File: Models/Data/Group.cs
using System.Collections.Generic;

namespace ohheck.help.Models.Data {
  public class Group : Common {
    public string name { get; set; }
    
    public virtual ICollection<Subunit> subunits { get; set; }
    public virtual ICollection<Idol> idols { get; set; }
  }
}

File: ohheck.help/src/types/admin/group.ts
import { ModelBase } from '../commontypes/modelbase';
import { Idol } from '../commontypes/idol';
import { Subunit } from './subunit';

export class Group extends ModelBase {
    name: string;
    subunits: Subunit[];
    idols: Idol[];

    constructor(json) {
        super(json);

        this.name = json.name;
        this.subunits = json.subunits;
        this.idols = json.idols;
    }
}